### DISCUSSION TITLE
Combo Meter & Easier To See Score

### DISCUSSION THREAD

**Discussion by ObsoleteGamer** [FunkinCrew/Funkin#1579] [Ideas]
Both ideas are tied to eachother:

Have it where a combo starts after 3 notes are hit in either "Sick" or "Cool", a combo around 3 - 30 combo give you 2x points multiplier with each point collected, your multiplier ends when you miss a note or hit it too late. 21 - 50 give you a 3x multiplier, 50 - 100 is 4x, and 100 - and beyond is x5. Your "Best Combo" can be viewed for each song and difficulty in story mode. Combo Chain and Multiplier carry over to the next song in the Week (it does not cross over when advancing to the next Week). Combos and points for each song in the week are counted individually and shown in the story mode menu when the Week is over.

The next idea is just to revamp the Score to be more visible to the player. It is very hard to see on Week 5 due to the snow.

**Comment by regulad** [comment]
I guess look at #249

**Comment by isakube** [comment]
imagine making combo-based scoring, br
